test/fixtures/jest/prefer-expect-assertions.fail.ts

```diff
@@ -0,0 +1,8 @@
+test('my test', () => {
+  expect.assertions('1');
+  expect(1).toEqual('foo');
+});
+
+test('my test', () => {
+  expect(1).toStrictEqual('foo');
+});
```

Comment:
these test names are the same, how does that even work? 🙈 

Also the test in the other file has `expect.hasAssertions();` Is this also a check that needs to be added to this test?

Reply:
> these test names are the same, how does that even work? 🙈

This is just a fixture, it does not really run those tests.

> Also the test in the other file has expect.hasAssertions(); Is this also a check that needs to be added to this test?

it is a valid case in for this rule so it will pass. see https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/prefer-expect-assertions.md

Reply:
AHHHHH... the test has nothing to do with `it` vs `test` then..... Then I would change them to both have `it`.

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/367/commits/0a24d089dcf2a71e426eff81252987b352d85cc4